wireapp/wire-android-reloaded
# fix: connections as conversations (AR-1813) (AR-1816)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1813" title="AR-1813" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-1813</a>  Pending connection request - Should stay in conversation list and behave as a conversation
  </td></table>
  <br />
 


















----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

This allows to shows connections and conversations in the same list, or as stated in bug raised and acceptance criteria, "connections should behave like a conversation".

This now consumes a new use case that returns connections and conversations in the same list.

### Dependencies (Optional)

https://github.com/wireapp/kalium/pull/628

Needs releases with: 


### Testing

#### Test Coverage (Optional)


#### How to Test

You can send connection request or receive from the app, the connections requests, either send or pending, should be listed in the conversations lists and move down accordingly to new updates are made to other conversations or new connections are created.

### Attachments (Optional)

https://user-images.githubusercontent.com/5806454/175956509-48d89f1a-b989-465f-8216-dc00389d08bd.mov

https://user-images.githubusercontent.com/5806454/175957345-24a409dc-e969-422d-bc10-0804847ad352.mov

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
